File: src/main/java/com/hirrao/xnntech/loader/RecipesLoader.java
package com.hirrao.xnntech.loader;

import com.hirrao.xnntech.recipes.CommonRecipes.CommonRecipes;
import com.hirrao.xnntech.recipes.CommonRecipes.MetaTileEntitiesCraftRecipes;
import com.hirrao.xnntech.recipes.MachineRecipes.MEInputRecipes;
import com.hirrao.xnntech.recipes.MachineRecipes.ProgrammableHatchesRecipes;

public class RecipesLoader {

    public static void load() {
        CommonRecipes.loadRecipes();
        MEInputRecipes.loadRecipes();
        ProgrammableHatchesRecipes.loadRecipes();
        MetaTileEntitiesCraftRecipes.loadRecipes();
    }

}

File: src/main/java/com/hirrao/xnntech/loader/RecipesLoader.kt
package com.hirrao.xnntech.loader

import com.hirrao.xnntech.recipes.commonRecipes.loadCommonRecipes
import com.hirrao.xnntech.recipes.commonRecipes.loadMTERecipes
import com.hirrao.xnntech.recipes.machineRecipes.loadMachineRecipes
import com.hirrao.xnntech.utils.Log

fun loadRecipes() {
    Log.info("Loading Recipes")
    loadCommonRecipes()
    loadMachineRecipes()
    loadMTERecipes()
}

